addons/loyalty/views/loyalty_reward_views.xml

```diff
@@ -17,7 +17,11 @@
                             <field name="reward_type" widget="selection" force_save="1" readonly="program_type == 'buy_x_get_y'"/>
                             <label for="discount" invisible="reward_type != 'discount'"/>
                             <div class="d-flex flex-row" invisible="reward_type != 'discount'">
-                                <field name="discount" class="oe_inline me-1"/>
+                                <field
+                                    name="discount"
+                                    class="oe_inline me-1 text-end"
+                                    style="max-width: 4rem;"
+                                />
```

Comment:
Do you really have to do this with the style attribute?
Can't you use standard bootstrap classes ? (`w-25` or sthg like that)

Reply:
As a div wraps the content that I wanted to limit in width, I can't define the width, only the max-width. And after some research in the bootstrap [documentation](https://getbootstrap.com/docs/5.3/utilities/sizing/), I found nothing to define a maximum width in rem, px, em, ... only percentage.